packages/presentation/lib/bloc/base/base_state.dart

```diff
@@ -0,0 +1,3 @@
+class BaseState {
+  const BaseState.init();
+}
```

Comment:
I think it should be an abstract class without constructor

Reply:
I created this class to init empty state in blocs where I don't have any states to avoid creating classes with empty constructors.